ooiui/templates/common/svgplot.html

```diff
@@ -81,7 +81,7 @@
             </div>
             <div class="panel-body">
               <div id="second-plot">
-                <i class="fa fa-spinner fa-spin" style="margin-left:50%;font-size:90px;"> </i>
+                <i class="fa fa-spinner fa-spin" style="margin-left:50%;font-size:90px;color:#337ab7"> </i>
```

Comment:
@ssontag55 we should make the color a CSS class so we can change them all at once